website/src/theme/DocPage/index.js

```diff
@@ -35,16 +35,19 @@ function DocPageContent({ currentDocRoute, versionMetadata, children }) {
     apisixHelmChart : "seeking-help",
     apisixDocker : "build-an-image-from-source",
     apisixJavaPluginRunner : "overview",
-    apisixGoRunner : "prerequisites"
+    apisixGoRunner : "prerequisites",
+    apisixFAQ: "why-a-new-api-gateway",
```

Comment:
Maybe not a good idea to use this way, because if the FAQ file gets updated, website will not know it should get updated, too.